spacy_llm/tasks/ner/registry.py

```diff
@@ -111,4 +116,66 @@ def make_ner_task_v2(
         case_sensitive_matching=case_sensitive_matching,
         single_match=single_match,
         scorer=scorer or score,
+        description=None,
+        check_label_consistency=check_label_consistency,
+    )
+
+
+@registry.llm_tasks("spacy.NER.v3")
+def make_ner_task_v3(
+    parse_responses: Optional[TaskResponseParser[SpanTask]] = None,
+    prompt_example_type: Optional[Type[FewshotExample]] = None,
+    labels: Union[List[str], str] = [],
+    template: str = DEFAULT_NER_TEMPLATE_V3,
+    label_definitions: Optional[Dict[str, str]] = None,
+    examples: ExamplesConfigType = None,
+    normalizer: Optional[Callable[[str], str]] = None,
+    alignment_mode: Literal["strict", "contract", "expand"] = "contract",
+    case_sensitive_matching: bool = False,
+    scorer: Optional[Scorer] = None,
+    description: Optional[str] = None,
+):
+    """NER.v3 task factory, with chain-of-thought prompting.
+
+    parse_responses (Optional[TaskResponseParser[SpanTask]]): Callable for parsing LLM responses for this task.
+    prompt_example_type (Optional[Type[FewshotExample]]): Type to use for fewshot examples.
+    labels (Union[str, List[str]]): List of labels to pass to the template,
+        either an actual list or a comma-separated string.
+        Leave empty to populate it at initialization time (only if examples are provided).
+    template (str): Prompt template passed to the model.
+    description (str): todo
```

Comment:
TODO

Reply:
Done in https://github.com/explosion/spacy-llm/pull/180/commits/3a31530bfd257cd77028a964e9b1e421676c0abd.